### PR TITLE
tiddler-body: wrap around floating elements

### PR DESCRIPTION
**Pull request by BurningTreeC**
found this solution here: http://maxdesign.com.au/articles/floatsample/

when an element within a tiddler is made floating this fix forces the tiddler body to wrap around it

### CODE REVIEW

**Comment by BurningTreeC** [comment]
@Jermolene , reopening this because it might be useful, especially for new users

File: themes/tiddlywiki/vanilla/base.tid [hunk 1]
```diff
@@ -1055,6 +1055,19 @@ canvas.tc-edit-bitmapeditor  {
...
+	content: "";
+	display: table;
+}
+ 
+.tc-tiddler-body:after {
+	clear: both;
+}
+ 
+.tc-tiddler-body {
+	zoom: 1;
```
Comment (pmario) [inline themes/tiddlywiki/vanilla/base.tid]: This is marked with IE6 / 7 ... which we can ignore, since TW won't work with them anyway.

**Comment by pmario** [comment]
see: https://css-tricks.com/snippets/css/clear-fix/

this should be enough. .... 

```
.clearfix:after {
  content: "";
  display: table;
  clear: both;
}
```

... but it's not usable for users. .... So we should create a proper clearfix class, that can be used everywhere. 

It would be needed, to assign the `clearfix` class to the tiddler body. Which means the ViewTemplate would be changed.

**Comment by pmario** [comment]
@BurningTreeC ... Why did you close it. I think it's worth fixing. ... but in a more generic way.

**Comment by BurningTreeC** [comment]
reopening...

@pmario, would you make the PR with your fix?

**Comment by BurningTreeC** [comment]
@pmario, do you want to add a fix for tiddler bodies not wrapping around floating elements?

**Pull request closed without merging**